spec/models/user_spec.rb

```diff
@@ -72,10 +72,15 @@
   end
 
   describe '#destroy' do
+    let(:user) { FactoryBot.create(:user) }
+
     context '主催するイベントが終了している場合' do
       it 'ユーザーの削除に成功すること' do
         travel_to '2000-08-01 00:00'.in_time_zone do
-          user = FactoryBot.create(:user, :with_finished_event)
+          FactoryBot.create(:event,
+            start_at: '2000-07-31 09:00',
```

Comment:
```suggestion
            start_at: '2000-07-31 09:00'.in_time_zone,
```

暗黙の型変換に依存するより、明示的にTime型に変えた方が思わぬトラブルが減ると思いますー。

Reply:
ありがとうございます！暗黙の型変換になっている箇所を修正しました！

該当コミット：https://github.com/ymmtd0x0b/awesome_events/pull/4/commits/445023d548d2b1bcc205d5cfe77e8f1245488789